crates/sui-verifier-transactional-tests/tests/id_leak/transmute.exp

```diff
@@ -2,4 +2,4 @@ processed 1 task
 
 task 0 'publish'. lines 4-28:
 Error: Transaction Effects Status: Sui Move Bytecode Verification Error. Please run the Sui Move Verifier for more information.
-Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: SuiMoveVerificationError, source: Some("ID is leaked into a struct.") } }
+Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: SuiMoveVerificationError, source: Some("ID is leaked into a struct. Found in _::m::transmute") } }
```

Comment:
Any idea why the package ID is always `_`?

Reply:
looks like the test runner maps addresses to a stable string for addresses. this feels familiar 😄. So anyway, it is a behavior specific to tests only

https://github.com/MystenLabs/sui/blob/c7f0c6186ad376f97fe95601a41bc841c90ba430/crates/sui-transactional-test-runner/src/test_adapter.rs#L660

https://github.com/MystenLabs/sui/blob/c7f0c6186ad376f97fe95601a41bc841c90ba430/crates/sui-transactional-test-runner/src/test_adapter.rs#L717

Reply:
All of the objects in these tests are randomly generated, so small I decided to hide addresses/object IDs in the outputs so the tests would be a bit more stable.